### PR TITLE
NN-2203 add redis config

### PR DESCRIPTION
**Pull request by JamieDraperUK**

### CODE REVIEW

File: helm_deploy/prison-staff-hub/templates/secrets.yaml [hunk 1]
```diff
@@ -11,4 +11,9 @@ data:
   API_SYSTEM_CLIENT_SECRET: {{ .Values.secrets.API_SYSTEM_CLIENT_SECRET | b64enc | quote }}
   GOOGLE_ANALYTICS_ID: {{ .Values.secrets.GOOGLE_ANALYTICS_ID | b64enc | quote }}
   SESSION_COOKIE_SECRET: {{ .Values.secrets.SESSION_COOKIE_SECRET | b64enc | quote }}
+  REDIS_HOST: {{ .Values.secrets.primary_endpoint_address | b64enc | quote }}
+  REDIS_AUTH_TOKEN: {{ .Values.secrets.auth_token | b64enc | quote }}
```
Comment (petergphillips) [inline helm_deploy/prison-staff-hub/templates/secrets.yaml]: these new secrets don't look like they have good names :disappointed:

Comment (JamieDraperUK) [reply helm_deploy/prison-staff-hub/templates/secrets.yaml]: No, they don't we can't choose them either.

Comment (petergphillips) [reply helm_deploy/prison-staff-hub/templates/secrets.yaml]: You'll need to change helm_deploy/prison-staff-hub/templates/_envs.tpl too

Comment (petergphillips) [reply helm_deploy/prison-staff-hub/templates/secrets.yaml]: Also secrets-example.yaml

**Pull request merged**